fastlane_core/lib/fastlane_core/build_watcher.rb

```diff
@@ -156,7 +156,11 @@ def processed?(build: nil, wait_for_build_beta_detail_processing: false)
         #
         # If set, this method will only return true if all three statuses are complete
         if wait_for_build_beta_detail_processing
-          is_processed &&= build.build_beta_detail.processed?
+          if build.build_beta_detail.nil?
+            is_processed = false
+          else
+            is_processed &&= build.build_beta_detail.processed?
```

Comment:
This is a great catch! I think this could also be reworked to something like this 👇 Thoughts? 😊 

`build&.build_beta_detail.processed?` will return `nil` if no `build_beta_detail` and then turn into `false` with `|| false`

```suggestion
          is_processed &&= (build&.build_beta_detail.processed? || false)
```

Reply:
Thanks for proposing safe navigation operator. @joshdholtz I was thinking if it's possible to do it without `if` statement ;)